mivs/config.py

```diff
@@ -4,3 +4,7 @@
 c.include_plugin_config(mivs_config)
 
 c.CODES_REQUIRING_INSTRUCTIONS = [getattr(c, code_type.upper()) for code_type in c.CODES_REQUIRING_INSTRUCTIONS]
+
```

Comment:
This is the first time we've run into this, but there are some new access levels which are only relevant to indie games.  In theory we could add those access levels to the main Uber config, but I'm experimenting with just keeping the plugins completely self-contained so that the core system doesn't need to do anything to add a plugin other than include it.